src/GitLink/Linker.cs

```diff
@@ -1,6 +1,6 @@
 ﻿// --------------------------------------------------------------------------------------------------------------------
-// <copyright file="Linker.cs" company="CatenaLogic">
-//   Copyright (c) 2014 - 2014 CatenaLogic. All rights reserved.
+// <copyright file="Linker.cs" company="Andrew Arnott">
```

Comment:
Not sure why this change is required? Not that I mind much, but we are considering moving to the .net foundation anyway. Just wondering if this is intended.